### ISSUE TITLE
Dwarf issue

### ISSUE THREAD

**Issue by KaylaHunther** [ZaneYork/SMAPI-Android-Installer#495]
after unlocking the dwarf (exploding the little blockage right of the mine entrance), for some reason the game is taking longer to load when I enter the mines, sometimes even crashing
did I make a mistake somewhere?

**Comment by Zencee** [comment]
you have to give more details like what mods you've been using that might affect your gameplay also check your logs when you enter the mine maybe an error occured.

**Comment by KaylaHunther** [comment]
> you have to give more details like what mods you've been using that might affect your gameplay also check your logs when you enter the mine maybe an error occured.

how do I check for error logs?

also I use the 2 cjb mods, OhoDavi's portrait mod, NPC map mod, platonic mod, 3 heart flower dance mod, some Sebastian mods like the dialogue expansion mod and yandere mod, slime hutch and slime hutch limit readjustment mod and a prismatic slime finder mod

**Comment by Zencee** [comment]
Follow the instructions for android then upload your log https://smapi.io/log

**Comment by Zencee** [comment]
to check your log in-game open up your virtual keyboard and press the "console" button you would know if theres an error since its stated in RED

**Comment by KaylaHunther** [comment]
there are no red texts, does that mean no errors? there are only grey, orange and yellowish text

**Comment by Zencee** [comment]
yep. Once you go to the mine you should check your logs at that time.

**Comment by KaylaHunther** [comment]
I found it! does this mean I should just disable one of them and I'm good to go?
![Uploading Screenshot_2023-07-17-21-28-34-95_b455cd2e68c7bcd1c56b48948a696206.jpg…]()

**Comment by KaylaHunther** [comment]
idk how to upload image sorry
but basically the logs said that two of my Sebastian mods are clashing. so does this mean I just need to disable one of them?

**Comment by Zencee** [comment]
you have to wait for the image to load before posting it.. anyways try and remove the one you've mentioned
